### PR TITLE
Improve UI design.

### PR DESCRIPTION
**Pull request by eatmore2004**
Implement `home` page functionality. Improve UI design.

### CODE REVIEW

File: apiClient/src/main/java/ua/nure/apiclient/service/ProductService.java [hunk 1]
```diff
@@ -76,4 +80,87 @@ public List<Product> getProductByBuyerId(String id) throws InterruptedException
...
+        String response = requestThread.getResponseToken();
+
+        if (!response.equals("true")) {
+            return Optional.empty();
+        }
+
+        return Optional.of(product);
+    }
+
+
```
Comment (NureBerehovyiDaniil) [inline apiClient/src/main/java/ua/nure/apiclient/service/ProductService.java]: Add more documentation

File: apiClient/src/main/java/ua/nure/apiclient/model/core/Product.java [hunk 2]
```diff
@@ -61,6 +65,29 @@ public Product(String productID, String cartID, String productName, String buyer
         this.endDate = endDate;
     }
```
Comment (NureBerehovyiDaniil) [inline apiClient/src/main/java/ua/nure/apiclient/model/core/Product.java:80]: Provide more info  about this constructor

**Review by NureBerehovyiDaniil — CHANGES_REQUESTED** [review]
@eatmore2004 See my comments

**Comment by eatmore2004** [comment]
@PetrPoling Please re-review.

**Review by NureBerehovyiDaniil — APPROVED** [review]
@eatmore2004 LGTM

**Pull request merged**